### PR TITLE
LineAreaGraph, RadialChart, LegendTable all updated

### PR DESCRIPTION
**Pull request by whitetiger1399**
Screenshot from 2021-02-18 14-59-47

Screenshot from 2021-02-18 14-23-01

Screenshot from 2021-02-18 14-23-08

### CODE REVIEW

File: src/lab/Graphs/LegendTable/LegendTable.tsx [hunk 1]
```diff
@@ -83,4 +72,17 @@ const LegendTable: React.FC<LegendProps> = ({
     </TableContainer>
   );
 };
+
+const LegendTable: React.FC<LegendTableProps> = ({ ...rest }) => {
+  return (
+    <ParentSize>
```
Comment (arkajyotiMukherjee) [inline src/lab/Graphs/LegendTable/LegendTable.tsx]: Can this be done with width: 100% in the LegendTable like we have discussed previously?

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: src/lab/Graphs/LegendTable/LegendTable.tsx (hunk 1 above)
Comment (whitetiger1399) [reply src/lab/Graphs/LegendTable/LegendTable.tsx]: sure

**Review by arkajyotiMukherjee — APPROVED** [review]
LGTM

**Review by S-ayanide — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]
lgtm

**Pull request merged**